app/frontend/store.js

```diff
@@ -30,6 +31,15 @@ const initialState = {
   validationMsg: undefined,
   authMethod: 'mnemonic',
   enableTrezor: CARDANOLITE_CONFIG.CARDANOLITE_ENABLE_TREZOR,
+  donateBtcUrl: CARDANOLITE_CONFIG.CARDANOLITE_DONATE_BTC_ADDRESS
+    ? BTC_BLOCKCHAIN_EXPLORER_URL + CARDANOLITE_CONFIG.CARDANOLITE_DONATE_BTC_ADDRESS
```

Comment:
I would leave out the ternary operator. I think it is safe to assume that the addresses would always be available